integrations/RollBar/browser.js

```diff
@@ -484,7 +484,7 @@ class RollBar {
     const { userId } = message;
     const { traits } = rudderElement.message.context;
 
-    var rollbar = window.Rollbar;
+    var rollbar = ;
```

Comment:
here rollbar points to nothing. Do we need this any more

Reply:
remove unused variables